src/fides/api/ops/task/graph_task.py

```diff
@@ -511,7 +511,12 @@ def access_request(self, *inputs: List[Row]) -> List[Row]:
         return filtered_output
 
     @retry(action_type=ActionType.erasure, default_return=0)
-    def erasure_request(self, retrieved_data: List[Row], *inputs: List[Row]) -> int:
+    def erasure_request(
+        self,
+        retrieved_data: List[Row],
+        inputs: List[List[Row]],
+        *_: int,
```

Comment:
Not sure the best name for this, this is just here so we can set the dependent tasks like we do for the access task graph

```
 saas_erasure_order_instance:labels: (<bound method GraphTask.erasure_request of <class 'fides.api.ops.task.graph_task.GraphTask'>:saas_erasure_order_instance:labels>,
                                      [{'labels_id': 1}],
                                      [[{'email': 'test@ethyca.com'}]],
                                      saas_erasure_order_instance:refunds,
                                      saas_erasure_order_instance:orders),
```

Reply:
`erasure_prereqs` sounds good